### DISCUSSION TITLE
Features vs Widgets

### DISCUSSION THREAD

**Discussion by dante01yoon** [feature-sliced/documentation#680] [Q&A]
Greetings, I see some examples' `features` layer has slices named with interactive prefix like `hover-book-list`, but some examples are not.

Can `features` slice have domain specific names without interactive naming convention like `book-list` ? 

Also, Is there any concrete guide where should I use features / widgets ?

**Comment by illright** [comment]
I'm not sure I fully understand the question. Features can be called anything, as long as they accurately describe the business purpose of the feature.

As for where to use features and widgets — I'd suggest to only start thinking about them when large blocks of UI need to be reused. Otherwise, it might be better to just keep it in the page

**Reply by dante01yoon** [reply]
If I understand correctly, there is no strict rule between widgets and features. Plz correct me if I 'm wrong. Thank you for respond
